src/nodes.coffee

```diff
@@ -270,7 +270,9 @@ exports.Base = class Base
   # as JSON. This is what the `ast` option in the Node API returns.
   # We try to follow the [Babel AST spec](https://github.com/babel/babel/blob/master/packages/babel-parser/ast/spec.md)
   # as closely as possible, for improved interoperability with other tools.
-  ast: (o) ->
+  ast: (o, level) ->
+    o = extend {}, o
```

Comment:
I've been trying to phase out these helpers when built-ins can do, like `Object.assign` in this case.

Reply:
Sure, updated to use `Object.assign()`